index.html

```diff
@@ -182,6 +182,21 @@ <h3 id="unacceptablebehaviour">
         <li>Retaliating against anyone who files a complaint that someone has violated this code of conduct.</li>
       </ul>
 
+      <p>The enforcers of this Code of Conduct should prioritise marginalized people’s safety over privileged people’s comfort,
+        and reserve the right not to act on complaints regarding:</p>
+
+      <ul>
+        <li>‘Reverse’ -isms, including ‘reverse racism,’ ‘reverse sexism,’ and ‘cisphobia’</li>
+
+        <li>Reasonable communication of boundaries, such as “leave me alone,” “go away,” or “I’m not discussing this with you.”</li>
+
+		<li>Refusal to explain or debate social justice concepts</li>
+		  
+        <li>Communicating in a ‘tone’ you don’t find congenial</li>
```

Comment:
Grammatical suggestion for consistency with the other bullets, and because I don't believe putting "tone" in quotes helps understanding:

> Communication in a tone not considered congenial.

Reply:
Thank you 👍

Reply:
Thanks for the update in 8cd6287 @AdaRoseCannon - did you miss my change of "Communicating" to "Communication"? It was part of the attempt towards grammatical consistency.

Reply:
Ah thanks will change now.